### PR TITLE
LRQA-53955 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-53955
AppBuilder#DeletePublishing

### CODE REVIEW

**Comment by barrosdennys** [comment]
Just started reviewing :)

File: modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro [hunk 1]
```diff
@@ -174,6 +188,16 @@ definition {
...
+	macro undeployApp {
+		Click(
+			key_itemName = "${itemName}",
+			locator1 = "AppBuilder#VERTICAL_ELLIPSIS_BUTTON");
+
+		MenuItem.clickNoError(menuItem = "Undeploy");
+
+		Click(locator1 = "AppBuilder#UNDEPLOY_APP_BUTTON");
+	}
+
```
Comment (barrosdennys) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro]: Hi @nkkwon ,
I was wondering here if we should have to validate the table status "Undeployed" as well, similar to what you are doing at the deleteApp, asserting that the element is no longer present)

What do you think?

@alexcandido10

**Comment by nkkwon** [comment]
Added validation for the Undeployed status! @barrosdennys

**Comment by nkkwon** [comment]
Should be good now!

**Comment by alexcandidos** [comment]
Hey @nkkwon , the commit still have a lot of commits not related to your changes, could you please send it to my branch instead of Dennys? I believe the Dennys branch was outdated/behind master when you pushed it. Sorry for that. :/

**Pull request closed without merging**